LICENSE-MIT

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2014 The Rust Project Developers
```

Comment:
Given this is embedded in Janus as if it's our license, it seems like the copyright here should reflect now and ISRG. We probably should check on this.